<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>180</y>
     <width>521</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>CALIBRATION</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="weightlabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="backbutton3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>400</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Save">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>400</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="clickherebutton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>310</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Click Here</string>
   </property>
  </widget>
  <widget class="QLabel" name="valuelabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid black;</string>
   </property>
   <property name="text">
    <string>Actual weight :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Place the weight on the load cell</string>
   </property>
  </widget>
  <widget class="QLabel" name="calibratelabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(173, 173, 173);</string>
   </property>
   <property name="text">
    <string>To calibrate the value</string>
   </property>
  </widget>
  <widget class="QLabel" name="op">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string> Device output status </string>
   </property>
  </widget>
  <widget class="QLabel" name="DOP1label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>DOP_01</string>
   </property>
  </widget>
  <widget class="QLabel" name="DOP2label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>90</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>DOP_02</string>
   </property>
  </widget>
  <widget class="QLabel" name="DOP3label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>DOP_03</string>
   </property>
  </widget>
  <widget class="QLabel" name="DOP4label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>130</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>DOP_04</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Device input  status</string>
   </property>
  </widget>
  <widget class="QLabel" name="DIP2label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgreen;</string>
   </property>
   <property name="text">
    <string>DIP_02</string>
   </property>
  </widget>
  <widget class="QLabel" name="DIP3label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>DIP_03</string>
   </property>
  </widget>
  <widget class="QLabel" name="DIP1label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>DIP_01</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clickedbutton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>100</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>clicked</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
